oak_runtime/src/node/http/tests.rs

```diff
@@ -23,14 +23,49 @@ use oak_abi::{
         NodeConfiguration,
     },
 };
-use std::{fs, option::Option, thread::JoinHandle};
+use std::fs;
 
 static LOCAL_CA: &str = "../examples/certs/local/ca.pem";
 static GCP_CA: &str = "../examples/certs/gcp/ca.pem";
 
+/// A simple Oak node that responds with 200 (OK) to every request it receives
+struct OakNodeForTest;
+
+impl crate::node::Node for OakNodeForTest {
+    fn node_type(&self) -> &'static str {
+        "test"
+    }
+    fn run(
+        self: Box<Self>,
+        runtime: RuntimeProxy,
+        handle: oak_abi::Handle,
+        _notify_receiver: oneshot::Receiver<()>,
+    ) {
+        // Get invocation message that contains the response_writer handle.
+        let invocation_receiver = Receiver::<HttpInvocation>::new(ReadHandle { handle });
+
+        while let Ok(invocation) = invocation_receiver.receive(&runtime) {
+            let request = invocation.receiver.unwrap().receive(&runtime);
+            info!("received request: {:?}", request);
```

Comment:
It may be interesting to do something with the request, at least to have some level of confidence it is received correctly, e.g. send back any headers and body, similar to an echo server. Otherwise we may again have issues in which we don't fully "look inside" what is coming in, and we risk missing catching bugs related to HTTP serialization.

Reply:
Done. It is now including the request body and the headers in the response.